### PR TITLE
irish pr day 8

### PR DESCRIPTION
**Pull request by IrishKnight0**

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_2.txt [hunk 1]
```diff
@@ -156,7 +156,7 @@ Yes, I have no opinion on who you choose to have a future relationship with, but
 [sha:8fe88f32d8e629f01c1284695efd2d5b499be606]{
 -- Page 66, Offset 19840.
 -- 「遠野家の長男が、その、どこの者とも判らない女性と夜な夜な逢引きをしているだなんて、そんな恥知らずな事はつつしむべきではないでしょうか？」
-"For the eldest son of the Tohno family to, well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
+"For the eldest son of the Tohno family to- well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
```
Comment (IrishKnight0) [inline script/Arcueid/Day 8/08_00_ARC08_2.txt:159]: I was thinking about this, I saw similar cases on the internet with a dash more than a coma

File: script/Arcueid/Day 8/08_00_ARC08_3.txt [hunk 2]
```diff
@@ -191,7 +191,7 @@ I knew I was having a hard time concentrating on the lesson,
 [sha:a7994a1cf038b4f576c81887fe127433df781fc3]{
 -- Page 150, Offset 20046.
 -- なにしろガラス越しに見下ろせる校庭に、優雅に歩いているあいつの幻が見えるなんて。
-I must be pretty out of it if I'm starting to hallucinate visions of that woman elegantly swanning across the school grounds.
+I must be pretty out of it if I'm starting to hallucinate visions of that woman, elegantly swanning across the school grounds.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_3.txt:194]: Not sure I agree with this one

File: script/Arcueid/Day 8/08_00_ARC08_3_1.txt [hunk 3]
```diff
@@ -137,13 +137,13 @@ Arcueid seems a little peeved,
...
 // Not sure who it makes sense to have be the subject of this first bit
-"'What are you trying to do?' That's my line!
+"'What are you trying to do?', that's my line!
 }
 [sha:dd37e49f4ac2129864647156b93fd6d1624b6a3e]{
 -- Page 169, Offset 20089.
 -- おまえ、歩く爆弾かなんかなのか!?」
 // Doesn't seem like a standard expression, literal it is then
-What are you, some sort of walking bomb?!"
+What are you, some sort of a walking bomb?!"
```
[the remark this answers is not in the document]
Comment (IrishKnight0) [reply script/Arcueid/Day 8/08_00_ARC08_3_1.txt:146]: should I remove the A in "Eh? Me? ...A walking bomb?"?

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_3_1.txt:146]: No, it's correct there

File: script/Arcueid/Day 8/08_00_ARC08_2.txt [hunk 4]
```diff
@@ -156,7 +156,7 @@ Yes, I have no opinion on who you choose to have a future relationship with, but
 [sha:8fe88f32d8e629f01c1284695efd2d5b499be606]{
 -- Page 66, Offset 19840.
 -- 「遠野家の長男が、その、どこの者とも判らない女性と夜な夜な逢引きをしているだなんて、そんな恥知らずな事はつつしむべきではないでしょうか？」
-"For the eldest son of the Tohno family to, well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
+"For the eldest son of the Tohno family to--- well, rendezvous with an unknown woman every night. Shouldn't you refrain from such indecent acts?"
```
Comment (IrishKnight0) [inline script/Arcueid/Day 8/08_00_ARC08_2.txt:159]: Script lint failed, revert?

**Comment by rschlaikjer** [comment]
Seems like a bug in the linter, will take a look this evening

File: script/Arcueid/Day 8/08_00_ARC08_2.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2.txt:159]: Ok, so what's happening here is that these are not the CJK dashes, they are normal dashes. And the parser sees the first two `--` and thinks ok, everything after this is a machine comment, I will just ignore it.

Replace them with proper CJK dashes and it'll fix the problem.

File: script/Arcueid/Day 8/08_00_ARC08_1_1.txt [hunk 5]
```diff
@@ -13,7 +13,7 @@ I feel like my usual approach isn't going to cut it here,
 -- Page 35, Offset 19761.
 -- 「グモニー。いい朝だね、秋葉」
 // Swap EN foreign loans for different foreign loans for effect?
-"Bonjour. Lovely morning isn't it, Akiha."
+"Bonjour. Isn't it a lovely morning, Akiha."
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_1_1.txt:16]: revert imo

**Review by Avuxo — APPROVED** [review]
lots of good catches here

**Pull request merged**